services/app/src/components/team/Team.jsx

```diff
@@ -25,7 +25,7 @@ function Team({ title, users, skipMove, children, removePlayer }) {
 
 Team.propTypes = {
   title: PropTypes.string.isRequired,
-  users: PropTypes.arrayOf(userPropType).isRequired,
+  players: PropTypes.arrayOf(userPropType).isRequired,
```

Comment:
If we are renaming `user` to `player`, we should also rename the `userPropType` accordingly.